src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const arrayPopulationStr = [
+  ...document.getElementsByClassName('population'),
+].map((n) => n.textContent);
+
+const arrayPopulationNum = arrayPopulationStr.map((str) => {
+  return Number(
+    str
+      .split('')
+      .filter((element) => element !== ',')
+      .join(''),
+  );
+});
+
+const total = arrayPopulationNum.reduce(
+  (accumulator, currentValue) => accumulator + currentValue,
+  0,
+);
+const average = total / arrayPopulationNum.length;
```

Comment:
The calculation of the average population is correct, dividing the total by the number of elements.